src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1014,6 +1014,11 @@ public sealed class ValidateRangeAttribute : ValidateEnumeratedArgumentsAttribut
         /// </summary>
         private Type _promotedType;
 
+        /// <summary>
+        /// Gets the name of the predefined range.
+        /// </summary>
+        internal ValidateRangeKind? RangeKind { get => _rangeKind; }
+
         ValidateRangeKind? _rangeKind;
```

Comment:
This should probably have `private` explicitly set, especially now that we're adding an `internal` accessor.